src/error.rs

```diff
@@ -45,6 +46,39 @@ pub enum SnafuError {
   },
   #[snafu(display("Unrecognized representation `{}`", input))]
   UnrecognizedRepresentation { source: error::Error, input: String },
+  #[snafu(display("Unrecognized outgoing amount: `{}`", input))]
+  UnrecognizedAmount {
+    source: bitcoin::amount::ParseAmountError,
+    input: String,
+  },
+  #[snafu(display("Unrecognized outgoing rune: `{}`", input))]
+  UnrecognizedRune {
+    source: ordinals::spaced_rune::Error,
+    input: String,
+  },
+  #[snafu(display("Unrecognized outgoing sat: `{}`", input))]
+  UnrecognizedSat {
+    source: ordinals::sat::Error,
+    input: String,
+  },
+  #[snafu(display("Unrecognized outgoing sat point: `{}`", input))]
+  UnrecognizedSatPoint {
+    source: ordinals::sat_point::Error,
+    input: String,
+  },
+  #[snafu(display("Unrecognized outgoing inscription ID: `{}`", input))]
+  UnrecognizedInscriptionId {
+    source: inscriptions::inscription_id::ParseError,
+    input: String,
+  },
+  #[snafu(display("Unrecognized outgoing: `{}`", input))]
+  UnrecognizedOutgoing { input: String },
+  #[snafu(display("Failed to parse decimal: {}", source))]
+  DecimalParse { source: error::Error, input: String },
```

Comment:
If this is only used in parsing rune amounts, then maybe `RuneAmountParse`.

Reply:
sure thing, pushing changes now.  Are you ok with having the following errors be so similar, or would you prefer to merge them?

OutgoingRuneParse and RuneParse
OutgoingSatParse and SatParse
OutgoingInscriptionIdParse and InscriptionIdParse

Reply:
Whoops, sorry, I wasn't looking at those other errors. Those errors can all be combined. In general, existing errors should be reused.

Reply:
ok, updates pushed.